Fix language and typos (#979)
grafana/grafana-infinity-datasource: docs/sources/json.md

Reviewer: More "we" instead of "you"

## Using public JSON API endpoints

> The example below shows fetching data from a publicly accessible JSON URL/API endpoint.

Reviewer: ```suggestion
The following example shows fetching data from a publicly accessible JSON API endpoint.
```

URL : `https://jsonplaceholder.typicode.com/users`

![image](https://user-images.githubusercontent.com/153843/108413678-34314c80-7223-11eb-9cce-603134ec2d48.png#center)

> In the example above, the data in the URL is an array, so there is no need to configure any additional fields except the URL in the panel.

Reviewer: ```suggestion
In the preceding example, the data in the URL is an array, so there is no need to configure any additional fields except the URL in the panel.
```

![image](https://user-images.githubusercontent.com/153843/108415716-cdf9f900-7225-11eb-8e0d-5d767104a080.png#center)

> In the example above, we're visualizing JSON data without time field. Our JSON has only two fields aka `country` and `population`, so we asked the plugin to add a dummy time field to the data so that we can visualize them in any of the Grafana's stock panels. If you look closely at the image above, you can see we specified 'format' as **timeseries**.

Reviewer: ```suggestion
In the example above, we're visualizing JSON data without time field. Our JSON has only two fields aka `country` and `population`, so we asked the plugin to add a dummy time field to the data so that we can visualize them in any of the stock panels in Grafana. If you look closely at the image above, you can see we specified 'format' as **timeseries**.
```

  { "country": "china", "population": 400 }
]
```
You also need to specify the column names manually for display purposes.

## JSONPath in root selector

In the root selector, you can use the selector in JSONPath format.

> Note: Any root selector that starts with $ will be considered as JSONPath selector.

Reviewer: ```suggestion
{{< admonition type="note" >}}
Any root selector that starts with `$` is considered as JSONPath selector.
{{< /admonition >}}
```

}
```

> In the above JSON, if `$.premium_customers` is the root selector then only "john doe" will return. If `$.*` is the root selector all the three rows will be returned.

Reviewer: ```suggestion
In the preceding JSON, if `$.premium_customers` is the root selector then the query returns `john doe`. If `$.*` is the root selector, the query returns all the three rows.
```
